content-src/experiments/tracking-protection.yaml

```diff
@@ -86,6 +86,7 @@ contributors:
     display_name: 'Rebecca Billings'
     title: 'Senior QA Engineer'
     avatar: /static/images/experiments/tracking-protection/avatars/rebecca-billings.jpg
+contributors_extra: 'This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years. Learn more about <a href="https://mzl.la/2fYneDq" target="blank">Firefox Tracking Protection</a>.'
```

Comment:
It's a bad practice, CC @flodolo to discuss further.

The right way to do it is to put the href on the element with the `data-l10n-id`, e.g.

```jsx
{contributors_extra && <p className="disclaimer"
    dangerouslySetInnerHTML={createMarkup(contributors_extra)}
    data-l10n-id={this.l10nId('contributors_extra')}
    href="https://whatever" />} 
```

Then in the string omit the argument entirely:

```
contributors_extra: 'This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years. Learn more about <a>Firefox Tracking Protection</a>.' 
```

l20n will handle moving the `href` attribute to the inner link for you when localizing the page. Unfortunately, we don't have a good system for handling that in experiment content right now.

If it's a 100% hard requirement that it be here, done in this way, feel free to dismiss my review and merge. I'm just reticent to approve it when it's such a hard hole to dig out of (see #1782, #1786, #1811).